UbiquityRobotics/magni_robot
# Adding fw_max_speed_fwd and /ubiquity_motor/fw_max_speed_rev to robot.yaml

Solving https://github.com/UbiquityRobotics/pi_image2/issues/39

To test:
```
cd ~/catkin_ws/src/
git clone https://github.com/UbiquityRobotics/magni_robot.git -b feature-newrobotyaml-params
cd ~/catkin_ws/
catkin_make -j2
source devel/setup.bash
```

add` fw_max_speed_fwd: 100 ` and `fw_max_speed_rev: -100` into `/etc/ubiquity/robot.yaml` under `ubiquity_motor`:
```
ubiquity_motor:
  fw_max_speed_fwd: 100
  fw_max_speed_rev: -100
```
then

`sudo systemctl restart magni-base.service` 

see if params have been set: 

```
rosparam get /ubiquity_motor/fw_max_speed_fwd
rosparam get /ubiquity_motor/fw_max_speed_rev
```
